Code/Comscore/Source/integration/Enums.swift

```diff
@@ -14,6 +14,24 @@ public enum ComScoreUserConsent: String {
     case unknown = "-1"
 }
 
+@frozen
+public enum ComscoreUsagePropertiesAutoUpdateMode: String {
+    case foregroundOnly
+    case foregroundAndBackground
+    case disabled
+    
+    func toComscore() -> SCORUsagePropertiesAutoUpdateMode {
+        switch self {
+        case .foregroundOnly:
+            return .foregroundOnly
```

Comment:
nit-pick: (maybe it is for me personally), but I think in this scenario maybe it would make it a bit more readable, if we use the class name too in the return?

```
        case .foregroundOnly:
            return SCORUsagePropertiesAutoUpdateMode.foregroundOnly
```

Reply:
✅